roles/docker/tasks/setup_apt.yml

```diff
@@ -39,7 +39,7 @@
   set_fact:
     docker_apt_release_channel: edge
   when:
-    - ansible_distribution == 'Debian' or ansible_distribution == 'Ubuntu'
+    - ansible_distribution == 'Debian' or ansible_distribution == 'Ubuntu' or ansible_distribution == 'Raspbian'
```

Comment:
Nice.
By the way, just an idea, given that we already have more than 2 options here, it could be nicer to use the following syntax:
```
"[ansible_distribution] | intersect(['Debian', 'Ubuntu', 'Raspbian']) | count"
```
I'd even go as far as to declare a list in global variables:
```
debian_family_list:
  - Debian
  - Ubuntu
  - Raspbian
```

Then all evaluations would look like:
```
when:
  - "[ansible_distribution] | intersect(debian_family_list) | count"
```